src/json-api/json-api-module.ts

```diff
@@ -391,4 +393,8 @@ export class JsonApiModule {
     };
   }
 
+  public toApiNotificationConfiguration(configuration: NotificationConfiguration): ApiNotificationConfiguration {
+    return configuration as ApiNotificationConfiguration;
```

Comment:
It is a conversion. The fact that the interfaces are actually identical and the conversion can be done like this is an implementation detail that shouldn't concern the consumers of this method. 

If they become unidentical at some point the conversion logic is placed here.

The fact that